src/pages/admin/CreateNewMeetingPage.vue

```diff
@@ -0,0 +1,122 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import apis, { CreateMeetingRequest } from '@/lib/apis'
+import BaseInput from '@/components/UI/BaseInput.vue'
+import BaseButton from '@/components/UI/BaseButton.vue'
+import EmoineHeader from '@/components/EmoineHeader.vue'
+import router from '@/router'
+
+//todo: 専用ヘッダー画像のアップロード
+const headerLogo = '/logo.png'
+
+const liveUrl = ref('')
+const isValidURL = ref(true)
+
+const getLiveIdFromUrl = (liveUrl: string) => {
+  const params = new URL(liveUrl).searchParams
+  if (!params.get('v')) throw new Error('Invalid URL')
+  return params.get('v')
+}
+
+const createMeeting = async () => {
+  let id = ''
+  try {
+    const videoId = getLiveIdFromUrl(liveUrl.value)
+    if (!videoId) throw new Error('Invalid URL')
+    const meetingInfo: CreateMeetingRequest = {
+      video_id: videoId,
+      description: ''
+    }
+    const res = (await apis.createMeeting(meetingInfo)).data
+    id = res.id
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      console.log(e.message)
+      isValidURL.value = false
+      return
+    }
+  }
+
+  // TODO: 遷移先の名前決定したらここに書く
+  router.push({ name: 'hoge', params: { id: id } })
+}
+</script>
+
+<template>
+  <emoine-header />
+  <div :class="$style.content">
+    <h2 :class="$style.title">
+      <img :src="headerLogo" />
+    </h2>
+    <div :class="$style.cotainer">
+      <div :class="$style.subTitleContainer">
+        <h3 :class="$style.subTitleText">LIVE URL</h3>
+        <div v-if="!isValidURL" :class="$style.errorText">
+          ※ 存在しないURLです
+        </div>
+      </div>
+      <div :class="$style.newEventInputCotainer">
+        <div :class="$style.input">
+          <base-input
+            :model-value="liveUrl"
```

Comment:
これでいい感じに動いた？
多分こうしないと動かない気がする
`v-model`っていうのはpropsとして`model-value`を渡すのと一緒に`update:modelValue`っていうイベントをコンポーネントから受け取ってて、どっちもやらないとbase-inputで入力した値が`liveUrl`の変数に入ってくれないと思う
```suggestion
            v-model="liveUrl"
```

つまり`v-model`で書き直したものは以下と同じ
```vue
<base-input
  :model-value="liveUrl"
  @update:model-value="liveUrl = $event"
  placeholder="YoutubeのライブURLを入力..."
 />
```